github/event_types.go

```diff
@@ -490,12 +490,11 @@ type PullRequestEvent struct {
 	PullRequest *PullRequest `json:"pull_request,omitempty"`
 
 	// The following fields are only populated by Webhook events.
-	Changes            *EditChange   `json:"changes,omitempty"`
-	RequestedReviewers []*User       `json:"requested_reviewers,omitempty"` // Populated in "review_requested", "review_request_removed" event deliveries.
-	Repo               *Repository   `json:"repository,omitempty"`
-	Sender             *User         `json:"sender,omitempty"`
-	Installation       *Installation `json:"installation,omitempty"`
-	Label              *Label        `json:"label,omitempty"` // Populated in "labeled" event deliveries.
+	Changes           *EditChange   `json:"changes,omitempty"`
+	RequestedReviewer *User         `json:"requested_reviewer,omitempty"` // Populated in "review_requested", "review_request_removed" event deliveries. A request for multiple users to review at once is split into multiple requests, each with a single, different RequestedReviewer.
```

Comment:
Would you mind moving this very long comment to the line above (on its own line) and wrap it reasonably to fit on a screen?

Reply:
moved to above and wrapped